cluster/gce/gci/mounter/stage-upload.sh

```diff
@@ -51,20 +48,8 @@ function cleanup {
 # Delete temporary directories on exit
 trap cleanup EXIT
 
-mkdir ${RKT_DIR}
 mkdir ${ACI_DIR}
 
-# Download rkt
```

Comment:
@jingxu97 is it ok to remove the rkt code from the gci mounter script?

Reply:
Ping @jingxu97

Reply:
Yes, I think so.